hooks/boot/01-helpers.rb

```diff
@@ -10,7 +10,7 @@ def module_enabled?(kafo, name)
       mod.enabled?
     end
 
-    def log_and_say(level, message)
+    def log_and_say(level, message, log_only = false)
```

Comment:
I noticed that later on it'd be a bit awkward. I'd at least prefer ```say=true``` rather than ```log_only=false``` for the parameter.